feature/interface/aggregate/otg_tests/aggregate_test/aggregate_test.go

```diff
@@ -227,6 +226,18 @@ func (tc *testCase) configureDUT(t *testing.T) {
 	tc.configDstAggregateDUT(agg, &dutDst)
 	aggPath := d.Interface(tc.aggID)
 	fptest.LogQuery(t, tc.aggID, aggPath.Config(), agg)
+
+	// Cleanup LACP and LAG configs if any to avoid potential conflicts
+	resetBatch := &gnmi.SetBatch{}
+	gnmi.BatchDelete(resetBatch, aggPath.Config())
+	gnmi.BatchDelete(resetBatch, lacpPath.Config())
+	resetBatch.Set(t, tc.dut)
+
+	// Apply LACP config only when LAG mode is LACP
+	if tc.lagType == lagTypeLACP {
+		gnmi.Replace(t, tc.dut, lacpPath.Config(), lacp)
+	}
+
```

Comment:
Earlier code always does a replace on both lacpPath.Config and aggPath.Config. even if tc.lagType =  lagTypeSTATIC.  Doing a gnmi.Replace should force an overwrite of any existing configuration.  It should not be necessary to delete configuration.    It seems if we are needing to delete configuration within these trees there is a bug?

Reply:
When lagType = lagTypeSTATIC, the interface should not be enabled under LACP hierarchy since LACP should not be run in that case. Previously, the script was enabling the interface under LACP for both lagTypeSTATIC and lagTypeLACP. The code change in this PR essentially is to achieve the below: start with a clean slate under both Interface_AGG and LACP hierarchies (by deleting the configs through gnmi.BatchDelete). Enable the interface under LACP only for lagType = lagTypeLACP.
Hope that clarifies.

Reply:
Isn't below block of code doing the same thing?

```
if deviations.AggregateAtomicUpdate(tc.dut) {
		tc.clearAggregate(t)
		tc.setupAggregateAtomically(t)
	}
```

Enabling `aggregate_atomic_update` deviation should suffice

Reply:
Just enabling the deviation wont help here since:
- clearAggregate() does not clear LACP configs
- the issue being addressed is not technically atomic vs non-atomic related. The issue at hand is that when LAG type is set to STATIC for an interface, the interface should be removed from under LACP.

Thanks!

Reply:
But the issue that I see here is that there shouldn't be any existing config.

```
aggID := netutil.NextAggregateInterface(t, dut)
```

Wouldn't this util always choose a non existing AggregateID?

Reply:
Anyways, for Arista we would need to move the below section such that it is called after the `delete`.

```
if deviations.AggregateAtomicUpdate(tc.dut) {
		tc.clearAggregate(t)
		tc.setupAggregateAtomically(t)
	}
```

Reply:
> But the issue that I see here is that there shouldn't be any existing config.
> 
> ```
> aggID := netutil.NextAggregateInterface(t, dut)
> ```
> 
> Wouldn't this util always choose a non existing AggregateID?

Yes, but after choosing a fresh aggID, we test lagTypeLACP and then lagTypeSTATIC on the same aggID.

Reply:
Made the change set simpler with the latest commit. LACP config will be applied only when lagType is LACP. Also the config delete is not needed in the middle if we test STATIC mode followed by LACP mode. Kindly review. Thanks!